src/mds/Server.cc

```diff
@@ -833,7 +833,7 @@ void Server::finish_flush_session(Session *session, version_t seq)
 
 void Server::_session_logged(Session *session, uint64_t state_seq, bool open, version_t pv,
 			     const interval_set<inodeno_t>& inos_to_free, version_t piv,
-			     const interval_set<inodeno_t>& inos_to_purge, LogSegment *ls)
+			     const interval_set<inodeno_t>& inos_to_purge, LogSegmentRef& ls)
```

Comment:
Where did you define `LogSegmentRef` ?

Reply:
In LogSegment.h after class LogSegment definition  
```
using LogSegmentRef = std::shared_ptr<LogSegment>;
```
Is it better to define it on another (generic) H file?